insert_gold_values.php

```diff
@@ -27,30 +27,35 @@ function fetchDailyMarketValuesJson() {
     return $data;
 }
 
-$daily_data = fetchDailyMarketValuesJson();
-$latest_sb_price = $daily_data[114]['price'];
+function setGoldValue($connection, int $item_id, int $gold_value) {
+    $insert_query = $connection->prepare('INSERT INTO item_markethunt (item_id, gold_value) 
+        VALUES (:item_id, :gold_value) ON DUPLICATE KEY UPDATE gold_value = VALUES(gold_value)');
+    $insert_query->execute([
+        'item_id' => $item_id,
+        'gold_value' => $gold_value,
+    ]);
+}
 
 setPDOConv();
 
-// insert/update gold values for items with markethunt entries
+$daily_data = fetchDailyMarketValuesJson();
+$latest_sb_price = $daily_data[114]['price'];
+
+// Insert/update gold values from markethunt
 foreach ($daily_data as $item_id => $item_data) {
     if ($item_data === NULL) continue;
 
-    $insert_query = $pdo_conv->prepare('INSERT INTO item_markethunt (item_id, gold_value) 
-        VALUES (:item_id, :gold_value)
-        ON DUPLICATE KEY UPDATE item_id = :item_id2, gold_value = :gold_value2');
-    $insert_query->execute([
-        'item_id' => $item_id,
-        'item_id2' => $item_id,
-        'gold_value' => $item_data['price'],
-        'gold_value2' => $item_data['price']
-    ]);
+    setGoldValue($pdo_conv, $item_id, $item_data['price']);
```

Comment:
this works. but for future, you don't have to pass pdo into a function. It is a global var, so you can just say `global $pdo_conv;` inside the function, and it will the the global var.